flutter/README.md

```diff
@@ -92,22 +77,23 @@ For a more throughout example see the [example](example/lib/main.dart).
 
 ##### Known limitations
 
-- We don't support the Flutter `split-debug-info` yet, if this feature is enabled, it'll give useless stack traces.
-- For the Flutter [obfuscate](https://flutter.dev/docs/deployment/obfuscate) feature, you'll need to upload the Debug symbols manually yet, See the section below.
-
-##### Debug Symbols for the Native integrations (Android and Apple)
+- Flutter `split-debug-info` and `obfuscate` flag aren't supported on iOS yet, but only on Android, if this feature is enabled, Dart stack traces are not human readable
+- If you enable the `split-debug-info` feature, you must upload the Debug Symbols manually.
 
-[Uploading Debug Symbols](https://docs.sentry.io/platforms/apple/dsym/) for Apple.
+##### Uploading Debug Symbols (Android and iOS)
 
-[Uploading Proguard Mappings and Debug Symbols](https://docs.sentry.io/platforms/android/proguard/) for Android.
+- [iOS dSYM files](https://docs.sentry.io/platforms/apple/dsym/)
+- [Android NDK](https://docs.sentry.io/product/cli/dif/#uploading-files), You've to do it manually, Do not use the `uploadNativeSymbols` flag from the [Sentry Gradle Plugin](https://docs.sentry.io/platforms/android/proguard/), it's not supported yet.
```

Comment:
```suggestion
- [Android NDK](https://docs.sentry.io/product/cli/dif/#uploading-files), You must to do it manually. Do not use the `uploadNativeSymbols` flag from the [Sentry Gradle Plugin](https://docs.sentry.io/platforms/android/proguard/), because it's not yet supported.
```